src/bootstrap/config.rs

```diff
@@ -1800,6 +1800,13 @@ impl Config {
         self.out.join(&*self.build.triple).join("ci-llvm")
     }
 
+    /// Directory where the extracted `rustc-dev` component is stored.
+    pub(crate) fn ci_rustc_dir(&self) -> PathBuf {
+        // assert!(self.download_rustc_commit.is_some());
```

Comment:
Why is this assert commented?

Reply:
oh, that's left over from before i switched it to `download_rustc()` instead of looking directly at the commit - i'll delete it. the difference between the two is that `download_rustc` will actually download artifacts which might be bad if we're calling it from within the download code, but i don't think it's ever called in that scenario.